For this document, I'm adopting the convention that $\text{Normal}(a, b)$ refers to the normal distribution with mean $a$ and variance $b$.

# Chapter 8.1

**2. Suppose that a random sample is to be taken from the normal distribution with unknown mean θ and standard deviation 2. How large a random sample must be taken in order that $E_θ(|\overline{X}_n − θ|^2) ≤ 0.1$ for every possible value of θ?**

The sampling distribution of the sample mean is given by,

$$
\overline{X}_n \sim \text{Normal}(\theta, {\sigma^2 \over n})
$$

Here, the standard deviation is known. So,

$$
\overline{X}_n \sim \text{Normal}(\theta, {4 \over n})
$$

Let,

$$
\epsilon = \overline{X}_n - \theta
$$

Then,

$$
\epsilon \sim \text{Normal}(0, {4 \over n})
$$

By the definition of variance,

$$
\operatorname{var}(\epsilon) = E[\epsilon^2] - E[\epsilon]^2
$$

The mean and variance of $\epsilon$ are known,

$$
{4 \over n} = E[\epsilon^2]
$$

Suppose,

$$
E[(\overline{X}_n - \theta)^2] \le 0.1
$$

Then,

$$
{4 \over n} \le 0.1
$$

$$
{4 \over 0.1} \le n
$$

$$
n \ge 40
$$

**9. Let $X_1, . . . , X_n$ be a random sample from the exponential distribution with parameter $θ$. Find the c.d.f. for the sampling distribution of the M.L.E. of $θ$. (The M.L.E. itself was found in Exercise 7 in Sec. 7.5.)**

We have that,

$$
X_1, \dots, X_n \sim \text{Exponential}(\theta)
$$

The MLE of $\theta$, $\hat{\theta}$ is given by,

$$
\hat{\theta} = {n \over \sum_{i = 1}^n X_i} = \overline{X}_n^{-1}
$$

We are interested in the cdf of $\hat{\theta}$. Let use denote this quantity with $F_{\hat{\theta}}$. THen,

$$
F_{\hat{\theta}}(\phi) = P(\hat{\theta} \lt \phi)
$$

$$
F_{\hat{\theta}}(\phi) = P\left({n \over \sum_{i = 1}^n X_i} \lt \phi \right)
$$

Assume $n \gt 0$, so,

$$
F_{\hat{\theta}}(\phi) = P\left({1 \over \sum_{i = 1}^n X_i} \lt {\phi \over n} \right)
$$

We have that $X_i \gt 0$, $\phi \gt 0$. So,

$$
F_{\hat{\theta}}(\phi) = P\left(\sum_{i = 1}^n X_i \gt {n \over \phi} \right)
$$

$$
F_{\hat{\theta}}(\phi) = 1 - P\left(\sum_{i = 1}^n X_i \le {n \over \phi} \right)
$$

The sum of $n$ exponentially distributed exponential variables with parameter $\theta$ has a gamma distribution with parameters $n$ and $\theta$. So,

$$
\sum_{i = 1}^n X_i \sim \text{Gamma}(n, \theta)
$$

Let $F_\gamma$ be the cdf of $\text{Gamma}(n, \theta)$. Then,

$$
F_{\hat{\theta}}(\phi) = 1 - F_\gamma \left({n \over \phi} \right)
$$

The cdf of the gamma distribution is analytically intractable. So, this is the simplified form.

# Chapter 8.2

**1. Suppose that we will sample 20 chunks of cheese in Example 8.2.3. Let $T = \sum_{i=1}^{20} (X_i− μ)^2/20$, where $X_i$ is the concentration of lactic acid in the ith chunk. Assume that $σ^2 = 0.09$. What number $c$ satisfies $Pr(T ≤ c)= 0.9$?**

We have that,

$$
T = \sum_{i = 1}^{20} (X_i - \mu)^2 /20
$$

$$
T = {\sigma^2 \over 20} \sum_{i = 1}^{20} {(X_i - \mu)^2 \over \sigma^2}
$$

But,

$$
{X_i - \mu \over \sigma} \sim \text{Normal}(0, 1)
$$

So,

$$
{(X_i - \mu)^2 \over \sigma^2} \sim \chi^2(1)
$$

So,

$$
\sum_{i = 1}^{20} {(X_i - \mu)^2 \over \sigma^2} \sim \chi^2(20) = \text{Gamma}(10, 1/2)
$$

Since we know $\sigma^2$, we can find,

$$
{\sigma^2 \over 20} = {0.09 \over 20} = 0.0045
$$

So,

$$
{\sigma^2 \over 20} \sum_{i = 1}^{20} {(X_i - \mu)^2 \over \sigma^2} \sim 0.0045 \text{Gamma}(10, 1/2)
$$

$$
{\sigma^2 \over 20} \sum_{i = 1}^{20} {(X_i - \mu)^2 \over \sigma^2} \sim \text{Gamma}(10, 1/((2)0.0045))
$$

$$
{\sigma^2 \over 20} \sum_{i = 1}^{20} {(X_i - \mu)^2 \over \sigma^2} \sim \text{Gamma}(10, 0.009)
$$

$$
T \sim \text{Gamma}(10, 0.009)
$$

We need the inverse CDF of $\text{Gamma}(10, 0.009)$ applied to 0.9.

In [1]:
from scipy import stats

stats.gamma(a = 10, scale = 1/0.009).ppf(0.9)

np.float64(1578.4433657947573)

So,

$$
c = 1578.4433657947573
$$

So that,

$$
P(T \le C) = 0.9
$$

**9. Suppose that $X_1, . . . , X_n$ form a random sample from the normal distribution with mean $\mu$ and variance $σ^2$. Find the distribution of,**

$$
{n(\overline{X}_n - \mu)^2 \over \sigma^2}
$$

The sample mean of a sample from the normal distribution has mean $\mu$ and variance $\sigma^2/n$.

$$
\overline{X}_n \sim \text{Normal}(\mu, \sigma^2/n)
$$

So,

$$
\overline{X}_n - \mu \sim \text{Normal}(0, \sigma^2/n)
$$

When you multiply a random variable by a scalar, the variance increases by the square of that scalar. For a symmetric distributions, multiplication by a scalar does not change the mean. So,

$$
{\overline{X}_n - \mu \over \sigma} \sim \text{Normal}(0, {1 \over \sigma^2}{\sigma^2 \over n})
$$

$$
{\overline{X}_n - \mu \over \sigma} \sim \text{Normal}(0, {1 \over n})
$$

So,

$$
\sqrt{n}{\overline{X}_n - \mu \over \sigma} \sim \text{Normal}(0, n{1 \over n})
$$

$$
\sqrt{n}{\overline{X}_n - \mu \over \sigma} \sim \text{Normal}(0, 1)
$$

The square of a standard normal variable is a $\chi^2(1)$. So,

$$
\left(\sqrt{n}{\overline{X}_n - \mu \over \sigma}\right)^2 \sim \chi^2(1)
$$

$$
n{(\overline{X}_n - \mu)^2 \over \sigma^2} \sim \chi^2(1)
$$

**10. Suppose that six random variables $X_1, . . . , X_6$ form a random sample from the standard normal distribution, and let,**

$$
Y = (X_1 + X_2 + X_3)^2 + (X_4 + X_5 + X_6)^2
$$

**Determine a value of $c$ such that the random variable $cY$ will have a $χ^2$ distribution.**

We have that,

$$
X_i \sim \text{Normal}(0, 1)
$$

The $X_i$ are normal, and the set of normals is closed under addition. So, the sum $X_1 + X_2 + X_3$ is normal. The expected value of the sum is the sum of the expectations. Since they are independent, the variance of the sum is the sum of the variances.

$$
X_1 + X_2 + X_3 \sim \text{Normal}(0 + 0 + 0, 1 + 1 + 1)
$$

So,

$$
X_1 + X_2 + X_3 \sim \text{Normal}(0, 3)
$$

When you multiply a random variable by a scalar, the variance is increased by a factor of the square of that scalar.

$$
{1 \over \sqrt{3}}(X_1 + X_2 + X_3) \sim \text{Normal}(0, \left({1 \over \sqrt{3}}\right)^23)
$$

So,

$$
{1 \over \sqrt{3}}(X_1 + X_2 + X_3) \sim \text{Normal}(0, 1)
$$

The square of the standard normal is the $\chi^2(1)$. So,

$$
\left({1 \over \sqrt{3}}(X_1 + X_2 + X_3)\right)^2 \sim \chi^2(1)
$$

$$
{1 \over 3}(X_1 + X_2 + X_3)^2 \sim \chi^2(1)
$$

Similarly,

$$
{1 \over 3}(X_4 + X_5 + X_6)^2 \sim \chi^2(1)
$$

The sum of two $\chi^2$ variables is distributed according to a $\chi^2$ with the sums of their degrees of freedom.

$$
{1 \over 3}(X_1 + X_2 + X_3)^2 + {1 \over 3}(X_4 + X_5 + X_6)^2 \sim \chi^2(2)
$$

$$
{1 \over 3}\left( (X_1 + X_2 + X_3)^2 + (X_4 + X_5 + X_6)^2 \right) \sim \chi^2(2)
$$

$$
{1 \over 3}Y \sim \chi^2(2)
$$

So, when $c = 1/3$, $cY$ is $\chi^2$ distributed.

# Chapter 8.3

**6. Suppose that $X_1, . . . , X_n$ form a random sample from the normal distribution with mean $μ$ and variance $σ^2$. Assuming that the sample size $n$ is 16, determine the values of the following probabilities:**

**(a) $P(\sigma^2{1 \over 2} \le {1 \over n}\sum_{i = 1}^n (X_i - \mu)^2 \le 2\sigma^2)$**

We have that,

$$
X_i \sim \text{Normal}(\mu, \sigma^2)
$$

Then,

$$
X_i - \mu \sim \text{Normal}(0, \sigma^2)
$$

$$
{X_i - \mu \over \sigma} \sim \text{Normal}(0, 1)
$$

$$
\left({X_i - \mu \over \sigma}\right)^2 \sim \chi^2(1)
$$

The sum of independent $\chi^2$ variables is a $\chi^2$ with the sum of the degrees of freedom.

$$
\sum_{i = 1}^n \left({X_i - \mu \over \sigma}\right)^2 \sim \chi^2(n)
$$

$$
{1 \over \sigma^2} \sum_{i = 1}^n \left(X_i - \mu\right)^2 \sim \chi^2(n)
$$

Assume $n \gt 0$,

$$
{n \over \sigma^2}{1 \over n} \sum_{i = 1}^n \left(X_i - \mu\right)^2 \sim \chi^2(n)
$$

Let,

$$
\hat{\sigma}^2 = {1 \over n} \sum_{i = 1}^n \left(X_i - \mu\right)^2
$$

Then,

$$
{n \over \sigma^2}\hat{\sigma}^2 \sim \chi^2(n)
$$

By assumption ${n \over \sigma^2} \gt 0$, so multiplication by this quantity constitutes a monotone increasing transform. So,

$$
P({\sigma^2 \over 2} \le \hat{\sigma}^2 \le 2\sigma^2) =
P({n \over \sigma^2}{\sigma^2 \over 2} \le {n \over \sigma^2}\hat{\sigma}^2 \le {n \over \sigma^2}2\sigma^2)
$$

$$
P({\sigma^2 \over 2} \le \hat{\sigma}^2 \le 2\sigma^2) =
P({n \over 2} \le {n \over \sigma^2}\hat{\sigma}^2 \le 2n)
$$

Let $\Phi_n$ denote the CDF of the $\chi^2(n)$. Then,

$$
P({\sigma^2 \over 2} \le \hat{\sigma}^2 \le 2\sigma^2) =
\Phi_n(2n) - \Phi_n(n/2)
$$

So,

$$
P(\sigma^2{1 \over 2} \le {1 \over n}\sum_{i = 1}^n (X_i - \mu)^2 \le 2\sigma^2) =
\Phi_n(2n) - \Phi_n(n/2)
$$

The $\chi^2$ CDF has no closed form, but we can evaluate this quantity numerically when $n = 16$.

In [2]:
n = 16
stats.chi2(df=n).cdf(2*n) - stats.chi2(df=n).cdf(n/2)

np.float64(0.9388666032540479)

So,

$$
P(\sigma^2{1 \over 2} \le {1 \over 16}\sum_{i = 1}^{16} (X_i - \mu)^2 \le 2\sigma^2) =
0.9388666032540479
$$

**(b) $P(\sigma^2{1 \over 2} \le {1 \over n}\sum_{i = 1}^n (X_i - \overline{X}_n)^2 \le 2\sigma^2)$**

The inner term is equal to the sample variance.

$$
\hat{\sigma}^2 = {1 \over n}\sum_{i = 1}^n (X_i - \overline{X}_n)^2
$$

Further, it is known that,

$$
{n \over \sigma^2}\hat{\sigma}^2 \sim \chi^2(n - 1)
$$

Consider the probability that this this term is between $n/2$ and $2n$, inclusive. Assume, $\sigma^2 \gt 0$ and $n \gt 0$. So,

$$
P({n \over 2} \le {n \over \sigma^2}\hat{\sigma}^2 \le 2n) = P({\sigma^2 \over 2} \le \hat{\sigma}^2 \le 2\sigma^2)
$$

Then,

$$
P({n \over 2} \le {1 \over \sigma^2}\hat{\sigma}^2 \le 2n) = P({1 \over 2}\sigma^2 \le {1 \over n}\sum_{i = 1}^n (X_i - \overline{X}_n)^2 \le 2\sigma^2)
$$

Let $\Phi_{\alpha}(x)$ denote the CDF of the $\chi^2$ distribution with $\alpha$ degrees of freedom. Then,

$$
P({n \over 2} \le {n \over \sigma^2}\hat{\sigma}^2 \le 2n) = \Phi_{n - 1}(2n) - \Phi_{n - 1}(n/2)
$$

Then,

$$
P({1 \over 2}\sigma^2 \le {1 \over n}\sum_{i = 1}^n (X_i - \overline{X}_n)^2 \le 2\sigma^2) = \Phi_{n - 1}(2n) - \Phi_{n - 1}(n/2)
$$

When $n = 16$,

$$
P({1 \over 2}\sigma^2 \le {1 \over n}\sum_{i = 1}^n (X_i - \overline{X}_n)^2 \le 2\sigma^2) = \Phi_{15}(2(16)) - \Phi_{16}((16)/2)
$$

$$
P({1 \over 2}\sigma^2 \le {1 \over 16}\sum_{i = 1}^{16} (X_i - \overline{X}_{16})^2 \le 2\sigma^2) = \Phi_{15}(32) - \Phi_{16}(8)
$$

There is no closed form for $\Phi_{15}$, but this term can be evaluated numerically.

In [3]:
stats.chi2.cdf(df=15, x=32) - stats.chi2.cdf(df=15, x=8)

np.float64(0.9173445555378668)

So, when $n = 16$,

$$
P(\sigma^2{1 \over 2} \le {1 \over n}\sum_{i = 1}^n (X_i - \overline{X}_n)^2 \le 2\sigma^2) = 0.9173445555378668
$$

We can interpret this as the probability that the sample variance is within a factor of 2 of the true variance. This is a small sample, so there is almost a 10% probability that the sample variance is off by a factor of at least 2.

**9. Suppose that each of two statisticians, $A$ and $B$, independently takes a random sample of 20 observations from the normal distribution with unknown mean $μ$ and known variance 4. Suppose also that statistician $A$ finds the sample variance in his random sample to be 3.8, and statistician $B$ finds the sample variance in her random sample to be 9.4. For which random sample is the sample mean likely to be closer to the unknown value of $μ$?**

One way to operationalize this question is to ask about the confidence intervals that $A$ and $B$ will have for $\mu$. In each case, the distribution of the sample mean is given by,

$$
\overline{X}_n \sim \text{Normal}(\mu, \sigma^2/n) = \text{Normal}(\mu, 0.2)
$$

Since the sampling distributions are the same, the confidence intervals will be the same. So, for example, the 95% confidence interval in each case is given by,

$$
\overline{X}_n \pm 1.96(0.2) = [\overline{X}_n - 0.392, \overline{X}_n + 0.392]
$$

Note that this only works because the variance is known. If the variance were unknown, then the sample variance would be used to estimate the variance for the confidence interval. In this case, $B$ would have a wider confidence interval.

Another way to operationalize this question would be to define some loss function $L$ and ask about the expected loss $A$ and $B$ would incur using their estimators, $\hat{\mu}_A$ and $\hat{\mu}_B$. Suppose the ratio is given by $c$, so that,

$$
E[L(\hat{\mu}_A, \mu)] = cE[L(\hat{\mu}_B, \mu)]
$$

If the variance is known, then,

$$
\hat{\mu}_A \sim \text{Normal}(\mu, 0.2)
$$

$$
\hat{\mu}_B \sim \text{Normal}(\mu, 0.2)
$$

So,

$$
E[L(\hat{\mu}_A, \mu)] = E[L(\hat{\mu}_B, \mu)]
$$

So,

$$
c = 1
$$

They have the same expected loss.